systemtest/edu/washington/escience/myria/systemtest/JsonOperatorTests.java

```diff
@@ -122,6 +123,6 @@ public void splitTest() throws Exception {
             .getProgramName(), outputRelation.getRelationName(), "json");
     String expectedData =
         "[{\"string_array\":\"a:b:c:d:e:f\",\"string_array_splits\":\"a\"},{\"string_array\":\"a:b:c:d:e:f\",\"string_array_splits\":\"b\"},{\"string_array\":\"a:b:c:d:e:f\",\"string_array_splits\":\"c\"},{\"string_array\":\"a:b:c:d:e:f\",\"string_array_splits\":\"d\"},{\"string_array\":\"a:b:c:d:e:f\",\"string_array_splits\":\"e\"},{\"string_array\":\"a:b:c:d:e:f\",\"string_array_splits\":\"f\"}]";
-    assertEquals(expectedData, data);
+    assertEquals(data, expectedData);
```

Comment:
The original order of args is correct: http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(java.lang.Object, java.lang.Object)